src/pages/Restaurants.jsx

```diff
@@ -142,6 +30,19 @@ const CardDescription = ({ description, rating, priceRange, cuisine }) => {
 }
 
 export const RestaurantsPage = () => {
+  const [cardsData, setCardsData] = useState()
```

Comment:
Please use an empty array as an initial state.
```suggestion
  const [cardsData, setCardsData] = useState([])
```

Reply:
Fixed